lib/eslint/flat-eslint.js

```diff
@@ -448,7 +447,7 @@ async function calculateConfigArray(eslint, {
  * @param {FlatConfigArray} config.configs The config.
  * @param {boolean} config.fix If `true` then it does fix.
  * @param {boolean} config.allowInlineConfig If `true` then it uses directive comments.
- * @param {boolean} config.reportUnusedDisableDirectives If `true` then it reports unused `eslint-disable` comments.
+ * @param {boolean|string} config.reportUnusedDisableDirectives If `true` or `error`, then it reports unused `eslint-disable` comments.
```

Comment:
If we're removing the constructor option `reportUnusedDisableDirectives`, this parameter should be removed too since it was passed the value of the constructor option.

Reply:
Fixed.